packages/api/lib/rulesHelpers.js

```diff
@@ -430,7 +473,7 @@ async function addSnsTrigger(item) {
       FunctionName: process.env.messageConsumer,
       Principal: 'sns.amazonaws.com',
       SourceArn: item.rule.value,
-      StatementId: `${item.name}Permission`,
+      StatementId: getSnsTriggerPermissionId(item),
```

Comment:
This could use another set of eyes. Originally this helper used `${item.name}Permission` here. This function is an existing (not new to this PR) duplicate of a method on the Rule model, `Rule.addSnsTrigger`. That method used `getSnsTriggerPermissionId`. 

I'm switching it here and updating the tests for this helper + integration tests for Rules(!). https://github.com/nasa/cumulus/pull/3149/files#diff-86dca067c5034695cd77c20c8de88c752c0f2b906583e669f203ae3f940f0dcaR119

I'm thinking that, so long as it's consistent, either approach works. The issue is that it wasn't consistent. Definitely need that validated though.

Reply:
Another, more compelling argument for using `getSnsTriggerPermissionId` is that's what is currently used in main:

https://github.com/nasa/cumulus/blob/master/packages/api/models/rules.js#L491

The rds-phase-3 branch is incorrect in having it set to `${item.name}Permission`. If we change the naming scheme users won't be able to update or remove existing permissions.

Reply:
Discussed in a huddle. The consensus is that `getSnsTriggerPermissionId` is correct here and that the feature/rds-phase-3 branch is incorrect.